client/src/pages/Login.js

```diff
@@ -0,0 +1,113 @@
+import React, { useState }  from "react";
+
+import { makeStyles, withStyles } from '@material-ui/core/styles';
+import { ValidatorForm, TextValidator } from 'react-material-ui-form-validator';
+import { Typography } from "@material-ui/core";
+import { Grid } from '@material-ui/core';
+import Button from '@material-ui/core/Button';
+import Container from '@material-ui/core/Container';
+
+const useStyles = makeStyles(theme => ({
+  textField: {
+    marginLeft: theme.spacing(1),
+    marginRight: theme.spacing(1),
+    boxShadow: "0px 0px 2px 0px rgb(175, 173, 179)",
+    borderRadius: 5,
+    width: "95%",
+    margin: 12,
+    textIndent: 10,
+    padding: 5,
+    height: 45,
+  },
+  button: {
+    margin: 40,
+  },
+  loginText: {
+    fontSize: 35,
+    color: "black",
+    textAlign: "center",
+    padding: 30,
+  }
+}));
+
+const SubmitButton = withStyles(theme => ({
```

Comment:
This type of component should be in it's own file in a different folder for reusable components

Reply:
Got it!